### DISCUSSION TITLE
Duda ejercicio 2.2

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#121] [Preguntas y respuestas] [answered]
Tengo una duda respecto a lo siguiente:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023590/d653cd14-8ece-494b-91d3-f2e9c4c361a4)
mi funcion retorna el largo de la cadena destino pero no se como o cuales serian los posibles codigos de errores ya que practicamente estas copiando una cadena.
https://github.com/INGCOM-UNRN-P1/tp3-2023-NicoHermosilla

**Comment by JeanFranco99** [comment, marked as answer]
-Argumentos invalidos
-Que la cadena destino no tenga el tamaño suficiente para contener la cadena 
-Que copie mas alla de los limites 
Son los que se me ocurren, capaz hay mas

**Reply by NicoHermosilla** [reply]
genial gracias!!

**Reply by martinvilu** [reply]
Marcá el comentario como respuesta porfis
